From 25d30b24dca1235c4e4fa06acd4fb98cf742cc75 Mon Sep 17 00:00:00 2001
From: Zhiyuan Chen <zhiyuanc@codeaurora.org>
Date: Thu, 15 Oct 2020 13:40:55 +0800
Subject: [PATCH] Resolve attribute issue on rmw native

Signed-off-by: Zhiyuan Chen <zhiyuanc@codeaurora.org>
---
 include/rmw/types.h | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/include/rmw/types.h b/include/rmw/types.h
index 32ed946..1c95b42 100644
--- a/include/rmw/types.h
+++ b/include/rmw/types.h
@@ -388,12 +388,6 @@ enum RMW_PUBLIC_TYPE rmw_qos_durability_policy_t
   "RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE is deprecated. " \
   "Use RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC if manually asserted liveliness is needed."
 
-#ifndef _WIN32
-# define RMW_DECLARE_DEPRECATED(name, msg) name __attribute__((deprecated(msg)))
-#else
-# define RMW_DECLARE_DEPRECATED(name, msg) name __pragma(deprecated(name))
-#endif
-
 /// QoS liveliness enumerations that describe a publisher's reporting policy for its alive status.
 /// For a subscriber, these are its requirements for its topic's publishers.
 // Suppress syntax errors, as cppcheck does not seem to handle enumerator attributes.
@@ -409,9 +403,7 @@ enum RMW_PUBLIC_TYPE rmw_qos_liveliness_policy_t
   /// Explicitly asserting node liveliness is required in this case.
   /// This option is deprecated, use RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC if your application
   /// requires to assert liveliness manually.
-  RMW_DECLARE_DEPRECATED(
-    RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE,
-    RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE_DEPRECATED_MSG) = 2,
+  RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE = 2,
 
   /// The signal that establishes a Topic is alive is at the Topic level. Only publishing a message
   /// on the Topic or an explicit signal from the application to assert liveliness on the Topic
-- 
2.7.4

