From 072c2f5925e10d90033ee8d4370a03398706d94f Mon Sep 17 00:00:00 2001
From: Bo Zhang <bozhang@codeaurora.org>
Date: Mon, 4 Mar 2019 15:17:27 +0800
Subject: [PATCH] remove HAS_LIBRARY_TARGET for ecl_devices

---
 ecl_devices/CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ecl_devices/CMakeLists.txt b/ecl_devices/CMakeLists.txt
index e69dad7..5ebd1d3 100644
--- a/ecl_devices/CMakeLists.txt
+++ b/ecl_devices/CMakeLists.txt
@@ -22,7 +22,6 @@ find_package(ecl_utilities REQUIRED)
 ##############################################################################
 # Project Configuration
 ##############################################################################
-
 ament_package_xml()
 ecl_enable_cxx14_compiler()
 
@@ -37,7 +36,7 @@ add_subdirectory(src)
 # Exports
 ##############################################################################
 
-ament_export_interfaces(HAS_LIBRARY_TARGET ${PROJECT_NAME})
+ament_export_interfaces(${PROJECT_NAME})
 ament_export_dependencies(
     ecl_config
     ecl_containers
-- 
1.9.1

