From 0848415f677ccd91857148ed2843999b3c4ed7d3 Mon Sep 17 00:00:00 2001
From: Bo Zhang <bozhang@codeaurora.org>
Date: Mon, 4 Mar 2019 15:30:32 +0800
Subject: [PATCH] remove HAS_LIBRARY_TARGET for ecl_geometry

---
 ecl_geometry/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ecl_geometry/CMakeLists.txt b/ecl_geometry/CMakeLists.txt
index 9825703..a9d8d30 100644
--- a/ecl_geometry/CMakeLists.txt
+++ b/ecl_geometry/CMakeLists.txt
@@ -38,7 +38,7 @@ add_subdirectory(src)
 # Exports
 ##############################################################################
 
-ament_export_interfaces(HAS_LIBRARY_TARGET ${PROJECT_NAME})
+ament_export_interfaces(${PROJECT_NAME})
 ament_export_dependencies(
     ecl_config
     ecl_containers
-- 
1.9.1

