From 9c7d6bcfa1be975764ac477107bebe773a763a96 Mon Sep 17 00:00:00 2001
From: Bo Zhang <bozhang@codeaurora.org>
Date: Mon, 4 Mar 2019 15:32:26 +0800
Subject: [PATCH] remove HAS_LIBRARY_TARGET for ecl_linear_algebra

---
 ecl_linear_algebra/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ecl_linear_algebra/CMakeLists.txt b/ecl_linear_algebra/CMakeLists.txt
index dd40d32..4a7024e 100644
--- a/ecl_linear_algebra/CMakeLists.txt
+++ b/ecl_linear_algebra/CMakeLists.txt
@@ -36,7 +36,7 @@ add_subdirectory(src)
 # Exports
 ##############################################################################
 
-ament_export_interfaces(HAS_LIBRARY_TARGET ${PROJECT_NAME})
+ament_export_interfaces(${PROJECT_NAME})
 ament_export_dependencies(
     ecl_eigen
     ecl_exceptions
-- 
1.9.1

