From 44c450af78850b10fc6e9d16cb5f6028a4735197 Mon Sep 17 00:00:00 2001
From: Bo Zhang <bozhang@codeaurora.org>
Date: Mon, 4 Mar 2019 15:23:13 +0800
Subject: [PATCH] remove HAS_LIBRARY_TARGET for ecl_exceptions

---
 ecl_exceptions/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ecl_exceptions/CMakeLists.txt b/ecl_exceptions/CMakeLists.txt
index 3215cff..77b47ba 100644
--- a/ecl_exceptions/CMakeLists.txt
+++ b/ecl_exceptions/CMakeLists.txt
@@ -32,7 +32,7 @@ add_subdirectory(src)
 # Exports
 ##############################################################################
 
-ament_export_interfaces(HAS_LIBRARY_TARGET ${PROJECT_NAME})
+ament_export_interfaces(${PROJECT_NAME})
 ament_export_dependencies(
     ecl_config
     ecl_errors
-- 
1.9.1

