From a69ef9a95429f114484fad2d547bf5177a602d3d Mon Sep 17 00:00:00 2001
From: Bo Zhang <bozhang@codeaurora.org>
Date: Mon, 4 Mar 2019 15:44:13 +0800
Subject: [PATCH] remove HAS_LIBRARY_TARGET for ecl_type_traits

---
 ecl_type_traits/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ecl_type_traits/CMakeLists.txt b/ecl_type_traits/CMakeLists.txt
index 8835fbd..29e3993 100644
--- a/ecl_type_traits/CMakeLists.txt
+++ b/ecl_type_traits/CMakeLists.txt
@@ -32,7 +32,7 @@ add_subdirectory(src)
 # Exports
 ###############################################################################
 
-ament_export_interfaces(HAS_LIBRARY_TARGET ${PROJECT_NAME})
+ament_export_interfaces(${PROJECT_NAME})
 ament_export_dependencies(
     ecl_config
     ecl_mpl
-- 
1.9.1

