From 6560f4f899a42c5af47752786c685cef0296466b Mon Sep 17 00:00:00 2001
From: Zhiyuan Chen <zhiyuanc@codeaurora.org>
Date: Fri, 13 Nov 2020 15:16:08 +0800
Subject: [PATCH] add back zstd module

Signed-off-by: Zhiyuan Chen <zhiyuanc@codeaurora.org>
---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5db06ce..05895ca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,7 +40,8 @@ target_include_directories(${PROJECT_NAME}_zstd
 ament_target_dependencies(${PROJECT_NAME}_zstd
   rcpputils
   rcutils
-  rosbag2_storage)
+  rosbag2_storage
+  ZSTD)
 # Causes the visibility macros to use dllexport rather than dllimport.
 # This is appropriate when building the dll but not consuming it.
 target_compile_definitions(${PROJECT_NAME}_zstd
-- 
2.7.4

