From 2d4c084bd1ed648c52d2adf0b1ca3404aa95260c Mon Sep 17 00:00:00 2001
From: Bo Zhang <bozhang@codeaurora.org>
Date: Mon, 4 Mar 2019 20:01:47 +0800
Subject: [PATCH] remove CTest

---
 CMakeLists.txt | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2665f7a..052532e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -96,12 +96,12 @@ target_compile_features (sophus INTERFACE
 add_custom_target(other SOURCES ${SOPHUS_OTHER_FILES})
 
 # Create 'test' make target using ctest
-include(CTest)
-option(BUILD_TESTS "Build tests." ON)
-if(BUILD_TESTS)
-    enable_testing()
-    add_subdirectory(test)
-endif()
+#include(CTest)
+#option(BUILD_TESTS "Build tests." ON)
+#if(BUILD_TESTS)
+#    enable_testing()
+#    add_subdirectory(test)
+#endif()
 
 # Export package for use from the build tree
 set(SOPHUS_CMAKE_EXPORT_DIR ${CMAKE_INSTALL_DATADIR}/sophus/cmake)
-- 
1.9.1

